### DISCUSSION TITLE
Contributor Graph

### DISCUSSION THREAD

**Discussion by TheColaber** [ScratchAddons/ScratchAddons#2276] [Show and tell]
I had some fun with gh's API xD

https://flo.uri.sh/visualisation/5950293/embed?auto=1

**Comment by cobaltt7** [comment]
Aw, it didn't go to now

**Reply by TheColaber** [reply]
wdym "to now"

**Reply by cobaltt7** [reply]
The graph stops when it's still pretty close to the beginning

**Reply by TheColaber** [reply]
??
![image](https://user-images.githubusercontent.com/72760579/115962444-71e37780-a4e9-11eb-8d7f-3bb4595e9a3c.png)
this is the present

**Reply by cobaltt7** [reply]
no?
![image](https://user-images.githubusercontent.com/61329810/115962855-ebc43280-a4e2-11eb-9a54-22f2f0e8b9ae.png) vs ![image](https://user-images.githubusercontent.com/61329810/115962867-f5e63100-a4e2-11eb-8f54-c8d21388be28.png)

**Reply by TheColaber** [reply]
Don't forget that the bot's commits were not included in the graph.

**Reply by cobaltt7** [reply]
ah ok

**Comment by cobaltt7** [comment]
i made a modified version
https://public.flourish.studio/visualisation/5954388/
